Wangscape/TilesetGenerator.cpp

```diff
@@ -28,34 +28,21 @@ void TilesetGenerator::generate(std::function<void(const sf::Texture&, std::stri
     boost::filesystem::path p(options.relativeOutputDirectory);
     for (const auto& clique : options.cliques)
     {
-        size_t res_x = options.resolution;
-        size_t res_y = options.resolution;
-        bool x_dimension = true;
-        for (int i = 0; i < CORNERS; i++)
-        {
-            size_t& res_z = x_dimension ? res_x : res_y;
-            res_z *= clique.size();
-            x_dimension = !x_dimension;
-        }
```

Comment:
Oh, didn't know. Wouldn't it be better to express that with proper enum? Something like:
```cpp
enum class Corners
{
    Triangle = 3,
    Square = 4,
    Hexagon = 6
};
```